styles.css

```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap');
+
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.container {
+  height: 500px;
+  background: white;
+  width: 375px;
+  background-repeat: no-repeat;
+  background-image:
+    url(./images/Ellipse\ 8.png),
+    url(./images/Rectangle\ 7.png),
+    url(./images/Ellipse\ 3.png);
+  background-position: center, left, right;
+}
```

Comment:
[OPTIONAL] Kindly make sure that your project is responsive. It should fit the container when you resize the screen. 
![image](https://user-images.githubusercontent.com/98363075/191969431-c8c1b5c6-7e7d-4974-b826-4e0c05b588b2.png)